storm-core/src/jvm/org/apache/storm/scheduler/SchedulerAssignment.java

```diff
@@ -55,4 +56,6 @@
     public Set<ExecutorDetails> getExecutors();
     
     public Set<WorkerSlot> getSlots();
+
+    public Map<WorkerSlot, Collection<ExecutorDetails>> getSlotToExecutors();
```

Comment:
I think we can just leave it for now and when we come up with a standard/style guide we can change it then